hack/local-up-cluster.sh

```diff
@@ -538,6 +539,11 @@ function start_kubelet {
         image_service_endpoint_args="--image-service-endpoint=${IMAGE_SERVICE_ENDPOINT}"
       fi
 
+      pod_infra_container_image=""
+      if [[ -n "${POD_INFRA_CONTAINER_IMAGE}" ]]; then
+        pod_infra_container_image="--pod_infra_container_image=${POD_INFRA_CONTAINER_IMAGE}"
```

Comment:
Please use dashes instead of underscores in our flags. Even though it accepts both, dashes are canonical.